lib/generators/avo/templates/initializer/avo.tt

```diff
@@ -6,10 +6,6 @@ Avo.configure do |config|
   # used only when you have custom `map` configuration in your config.ru
   # config.prefix_path = "/internal"
 
-  # Sometimes you might want to mount Avo's engines yourself.
-  # https://docs.avohq.io/3.0/routing.html
-  # config.mount_avo_engines = true
```

Comment:
This should still be here with a warning that it's not working anymore.
Deprecation warning.

Maybe